python/lsst/sims/photUtils/Photometry.py

```diff
@@ -569,7 +569,8 @@ def sum_magnitudes(self, disk = None, bulge = None, agn = None):
             raise RuntimeError("Arguments of sum_magnitudes need to be " +
                                "either floats or numpy arrays; you appear to have passed %s " % baselineType)
 
-        mm_o = 22.
+        mm_0 = 22.
```

Comment:
Can you document where this zero-point magnitude value comes from? According to this: https://www.sdss3.org/dr8/algorithms/magnitudes.php I see this should maybe be 22.5?